README.md

```diff
@@ -125,6 +125,19 @@ fetch('http://httpbin.org/post', { method: 'POST', body: stream })
 		console.log(json);
 	});
 
+// post with JavaScript object
+
+var body = { a: 1 };
+fetch('http://httpbin.org/post', { 
+        method: 'POST',
+        body:    JSON.stringify(body),
+        headers: { 'Content-Type': 'application/json' },
+    }).then(function(res) {
+        return res.json();
+    }).then(function(json) {
+        console.log(json);
+    });
```

Comment:
Thanks for giving us `fetch` on Node, by the way – I've dispensed with jQuery `$.ajax` client-side, and now can use the same api server-side :)